### PR TITLE
AB#7018 sitecore send content json added to demo folder

### PR DESCRIPTION
**Pull request by mintsitecore**

### CODE REVIEW

File: demo/send/mailingLists/7dc7a9bd-0a14-427f-894a-66d8f7a938db.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"Campaign":null,"SendToDetails":null,"Content":null,"OptIn":0,"SegmentsCount":0,"LastDayChanges":0,"ActiveSubscribers":5,"MailingList":{"Id":"7dc7a9bd-0a14-427f-894a-66d8f7a938db","MailingListCode":"2930539","Name":"first mailing list","MailingListName":"first mailing list","ActiveMemberCount":5,"BouncedMemberCount":0,"RemovedMemberCount":0,"UnsubscribedMemberCount":0,"Status":0,"CreatedBy":"7dc7a9bd0a14427f894a66d8f7a938db","CreatedOn":"/Date(1641564378217-0000)/","UpdatedOn":"/Date(1658417249577-0000)/","DeletedMemberCount":0,"IsSupressionList":false},"Entity":null}
```
Comment (jflheureux) [inline demo/send/mailingLists/7dc7a9bd-0a14-427f-894a-66d8f7a938db.json:1]: is this "first mailing list" really used by the solution and must be serialized here? Should it be renamed to something more meaningful if it is being used?

File: demo/send/mailingLists/3c869606-e1d4-4d37-9267-5e38bc6efeb8.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"Campaign":null,"SendToDetails":null,"Content":null,"OptIn":0,"SegmentsCount":0,"LastDayChanges":0,"ActiveSubscribers":2,"MailingList":{"Id":"3c869606-e1d4-4d37-9267-5e38bc6efeb8","MailingListCode":"3050944","Name":"Generated for Email Activity","MailingListName":"Generated for Email Activity","ActiveMemberCount":2,"BouncedMemberCount":0,"RemovedMemberCount":0,"UnsubscribedMemberCount":0,"Status":0,"CreatedBy":"7dc7a9bd0a14427f894a66d8f7a938db","CreatedOn":"/Date(1658175357343-0000)/","UpdatedOn":"/Date(1658841548140-0000)/","DeletedMemberCount":0,"IsSupressionList":false},"Entity":null}
```
Comment (jflheureux) [inline demo/send/mailingLists/3c869606-e1d4-4d37-9267-5e38bc6efeb8.json:1]: is this "Generated for Email Activity" really used by the solution and must be serialized here? Should it be renamed to something more meaningful if it is being used?

File: demo/send/mailingLists/27cda576-4a60-4621-b515-8489b396a300.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"Campaign":null,"SendToDetails":null,"Content":null,"OptIn":0,"SegmentsCount":0,"LastDayChanges":0,"ActiveSubscribers":2,"MailingList":{"Id":"27cda576-4a60-4621-b515-8489b396a300","MailingListCode":"2947879","Name":"Subscribers from https://adotest-cm.sitecoredemo.com/","MailingListName":"Subscribers from https://adotest-cm.sitecoredemo.com/","ActiveMemberCount":2,"BouncedMemberCount":0,"RemovedMemberCount":0,"UnsubscribedMemberCount":0,"Status":0,"CreatedBy":"7dc7a9bd0a14427f894a66d8f7a938db","CreatedOn":"/Date(1643747091280-0000)/","UpdatedOn":"/Date(-62135596800000-0000)/","DeletedMemberCount":1,"IsSupressionList":false},"Entity":null}
```
Comment (jflheureux) [inline demo/send/mailingLists/27cda576-4a60-4621-b515-8489b396a300.json:1]: is this "Subscribers from https://adotest-cm.sitecoredemo.com/" really used by the solution and must be serialized here? Should it be renamed to something more meaningful if it is being used?

File: demo/send/landingPages/9ed4f663-8687-4df2-8f35-a201231312b4.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"Campaign":null,"SendToDetails":null,"Content":null,"OptIn":0,"SegmentsCount":0,"LastDayChanges":0,"ActiveSubscribers":0,"MailingList":null,"Entity":{"ConversionPercentage":0,"CreatedBy":"129.222.164.164","CreatedOn":"/Date(1643743574790-0000)/","Id":"9ed4f663-8687-4df2-8f35-a201231312b4","Name":"jf-test","PublishContext":"http://sitecoresendemo.sitecoresend.net/syPFQZ/jf-test","PublishContextNoProtocol":"sitecoresendemo.sitecoresend.net/syPFQZ/jf-test","PublishMode":1,"Status":0,"TotalConversions":0,"TotalPageViews":0,"TotalUniqueVisits":0,"Type":2,"sUpdatedBy":null,"UpdatedOn":"/Date(1658433456153-0000)/","UserId":"7dc7a9bd-0a14-427f-894a-66d8f7a938db","WebsiteId":"9ed4f663-8687-4df2-8f35-a201231312b4"}}
```
Comment (jflheureux) [inline demo/send/landingPages/9ed4f663-8687-4df2-8f35-a201231312b4.json:1]: is this "jf-test" really used by the solution and must be serialized here? Should it be renamed to something more meaningful if it is being used?

File: demo/send/campaigns/4ff6e5fe-3560-40c0-abf0-7834ed05de3d.json [hunk 5]
```diff
@@ -0,0 +1 @@
...\"body\":\"// This code will be added to a <Script> tag, \\n// at the end of the <body> element \\n\"},\"primaryProduct\":{\"sync\":false,\"status\":false,\"description\":\"\",\"startDate\":\"\",\"endDate\":\"\",\"couponCode\":\"\",\"src\":\"\",\"resized\":false,\"resizeWidth\":null,\"resizeHeight\":null,\"currentSrc\":\"\",\"originalSrc\":\"\"},\"watermark\":{\"style\":\"white-left\",\"alignment\":\"center\",\"padding\":20},\"templateId\":\"2caeacb0-6aab-4167-9b8d-24c77f4757b8\",\"hasRepeatables\":0,\"repeatableRowId\":\"\"}}","Plain":" SAMPLEACTIVITYHave yourself a merry little present (on us)Have yourself a merry little present (on us)This Christmas a special gift awaits you. Browse our wide range of holiday products and choose something to add to your cart. Your special freebie will be there too!This Christmas a special gift awaits you. Browse our wide range of holiday products and choose something to add to your cart. Your special freebie will be there too!Grab your present!Louder180 A32nd StreetBronx, New YorkWant to stop receiving emails from us? You can unsubscribe here "},"OptIn":0,"SegmentsCount":0,"LastDayChanges":0,"ActiveSubscribers":0,"MailingList":null,"Entity":null}
```
Comment (jflheureux) [inline demo/send/campaigns/4ff6e5fe-3560-40c0-abf0-7834ed05de3d.json:1]: is this "Generated for Email Activity" really used by the solution and must be serialized here? Should it be renamed to something more meaningful if it is being used?

**Review by jflheureux — CHANGES_REQUESTED** [review]
Please review all the serialized automations, campaigns, landing pages, and mailing lists. Identify those that are really used. Delete the ones that are unused.

**Comment by adoprog** [comment]
We've added this content (and more) to portal provisioning.

**Pull request closed without merging**